src/utils/configuration-reader.js

```diff
@@ -0,0 +1,60 @@
+import yargs from 'yargs';
+import cosmiconfig from 'cosmiconfig';
+import camelcase from 'camelcase';
+import ConfigurationParsingException from './exceptions/configuration-parsing-exception';
+
+function parseCsvArray(value) {
+  return (value || '').split(',');
+}
+
+export function getCliArgs(args, version) {
+  return yargs(args)
+    .usage('Usage: $0 [DIRECTORY]')
+    .boolean(['dev', 'ignore-bin-package', 'skip-missing'])
+    .default({
+      dev: true,
+      'ignore-bin-package': false,
+      'skip-missing': false,
```

Comment:
If we set a default in the CLI parameters here, we will never be able to set the values from the configuration file.

Fix: the default values of the CLI parameters should be removed, they're not necessary.

Reply:
`defaultOptions` has the same purpose later and it's great.